app/views/pages/job-application-preview.html.slim

```diff
@@ -0,0 +1,211 @@
+- content_for :page_title_prefix do
+  | Job Application Preview
+
+.govuk-main-wrapper
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      h2.govuk-heading-l id="heading" Teaching Vacancies application form
```

Comment:
You should use default slim syntax for adding ids:
`h2#heading.govuk-heading-l Teaching Vacancies application form`
But why do you give it an id with name `heading`? it seems more a class for me

Reply:
Sure, I'll do that now. I should've remember this as we discussed it recently!

The id 'heading' was added for the 'back to top' anchor links.